example/tests/Drupal/Nightwatch/Tests/exampleTest.js

```diff
@@ -12,6 +12,6 @@ module.exports = {
       .drupalRelativeURL('/')
       .waitForElementVisible('body', 1000)
       .assert.containsText('body', 'Log in')
-      .drupalLogAndEnd({ onlyOnError: false });
+      .drupalLogAndEnd({ onlyOnError: "FALSE" });
```

Comment:
Is this change related to this PR, or did it slip in? If it was necessary for tests to pass or something, that's fine. Otherwise, let's pull it out of this pull request and you can explain the motivation to me to consider separately. I'll go ahead and approve the PR and trust you to use your judgment as to whether to change it. 👍

Reply:
@TravisCarden This change is enforced by the latest version of _acquia/coding-standards_ , thus it is required as a part of this PR for tests to pass.